apps/mobile/src/features/approval-ux/approval.tsx

```diff
@@ -0,0 +1,54 @@
+import { useState } from 'react';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+
+import { BottomSheetScrollView } from '@gorhom/bottom-sheet';
+import { t } from '@lingui/macro';
+import { useTheme } from '@shopify/restyle';
+
+import { Box, Text, Theme } from '@leather.io/ui/native';
+
+import { AdvancedDetailsButton } from './components/advanced-details-button';
+import { AccountCard } from './components/cards/account-card';
+import { CodeCard } from './components/cards/code-card';
+import { FeeCard } from './components/cards/fee-card';
+import { InputsAndOutputsCard } from './components/cards/inputs-outputs-card';
+import { OutcomesCard } from './components/cards/outcomes-card';
+import { Footer } from './components/footer';
```

Comment:
The core styles for approver ux should be in UI package `src/components/approver`. 

Also approver should not be "aware" of any details like fees/inputs etc. `signPsbt` should be an instance of `<Approver />`